### ISSUE TITLE
When saving Schedules Direct

### ISSUE THREAD

**Issue by 12nick12** [jellyfin/jellyfin#2628]
When saving Schedules Direct and I hit save all of the data goes back to how it was before I hit save. This is on Debian 10 and using a HDHomeRun Quatro as the tuner. The tuner plays fine, but the schedules don't want to work for some reason.

**Comment by zehnerGIT** [comment]
known bug:
https://forum.jellyfin.org/t/network-settings-do-not-stick/1454
https://github.com/jellyfin/jellyfin-plugin-template/issues/8
https://github.com/jellyfin/jellyfin-plugin-ldapauth/issues/21
...

your values should be saved (check the xml with the settings in your plugin directory), but the web shows the old values

**Comment by 12nick12** [comment]
Ah ok. Thank you. On Mar 19, 2020 9:28 AM, zehner <notifications@github.com> wrote:
known bug:
https://forum.jellyfin.org/t/network-settings-do-not-stick/1454
jellyfin/jellyfin-plugin-template#8
jellyfin/jellyfin-plugin-ldapauth#21
...
your values should be saved (check the xml with the settings in your plugin directory), but the web shows the old values

**Comment by ChaddyOwl** [comment]
I recently experienced something similar. After configuring Schedules Direct, and hitting the Save button to login, the details that I selected under Select Listings (Country, Zip Code, and Lineup) cannot be saved. There is no Save button beneath these, and pressing the Save button under the Schedules direct login data just clears the form. After doing this, not guide updates would appear.
I had an older Config directory backed up, and inside the `livetv.xml` file, I could see that the three parameters `ListingsId`, `ZipCode` and `Country` were missing from the `ListingsProviderInfo` section. After I restored those lines in my latest `livetv.xml` file , and restarted the server, everything worked normally once again.
I only installed my first Jellyfin server a few weeks ago, and only recently updated to 10.5.0. This worked without any issues when I first installed this, so it feels like a recently introduced issue.
